apis/v1beta1/validation/httproute.go

```diff
@@ -334,6 +341,28 @@ func validateHTTPHeaderModifier(filter gatewayv1b1.HTTPHeaderFilter, path *field
 	return errs
 }
 
+func validateHTTPRouteTimeouts(timeouts *gatewayv1b1.HTTPRouteTimeouts, path *field.Path) field.ErrorList {
```

Comment:
This still needs a unit test in httproute_test.go?

Reply:
addressed in 0fb746498cddabddee5c70bb6d377ac163f10999